src/axes.jl

```diff
@@ -81,6 +81,7 @@ end
 function attr!(axis::Axis, args...; kw...)
     # first process args
     plotattributes = axis.plotattributes
+    dummy_attributes = Dict{Symbol, Any}(:plot_object=>first(axis.sps).plt)
```

Comment:
What are those needed for?

Reply:
Good question. `RecipesPipeline._apply_type_recipe` has
https://github.com/JuliaPlots/Plots.jl/blob/399ee6a415dde2b3339f44cbc09e1eab616d5dd4/RecipesPipeline/src/type_recipe.jl#L18-L25
(does Github show a summary?)
where it asks the first argument for `:plot_object`, which it then uses for pre- and postprocessing. Perhaps the function should instead be extended to accept an `Axis` object instead or something, this just seemed like the minimum intrusion operation to me.